src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -222,10 +223,15 @@ HttpStatusCode mongoSubscribeContext
 
     LM_T(LmtSubCache, ("inserting a new sub in cache (%s)", oidString.c_str()));
 
+    // FIXME Px: In the near future, the subscription carries the HttpInfo
```

Comment:
ok, you mean that with:

    httpInfo.url = requestP->reference.get();
    
    if (apiVersion == "v2")
    {
        httpInfo.xxx = yyy;
        etc
    }

... everything is clear enough ...

Howver, what I meant with this comment is that right now, the HttpInfo is 'simulated', instead of being input to this function.
Are we on the same page?